### PR TITLE
feat(public-cloud): update fr content

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-localzone` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-12742 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by anooparveti — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Review by dectotam — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/project.service.js [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
     return this.project;
   }
 
+  getDocUrl(SUFFIX) {
+    const suffix = URL_INFO[SUFFIX];
+    return `${URL_INFO.PREFIX}/${this.coreConfig
+      .getUserLocale()
+      .toLowerCase()
+      .replace('_', '-')}/${suffix}`;
+  }
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/project.service.js:22]: **question:** Why are we using locale? Should not it be subsidiary?

File: packages/manager/modules/pci/src/projects/project/project.constants.js [hunk 2]
```diff
@@ -172,6 +172,12 @@ export const LEGACY_PLAN_CODES = ['project.legacy', 'project.2018'];
 
 export const LOCAL_ZONE_REGION = 'localzone';
 
+export const URL_INFO = {
+  PREFIX: 'https://ovhcloud.com',
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/project.constants.js:176]: **question:** Is it always the same for all regions?

File: packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html [hunk 3]
```diff
@@ -103,14 +103,68 @@
...
+                                        data-translate="pci_project_flavors_zone_localzone_tooltip"
+                                    ></p>
+                                    <a
+                                        class="fa fa-external-link ml-2"
+                                        aria-hidden="true"
+                                        data-ng-href="{{$ctrl.localZoneUrl}}"
+                                        title="{{ 'pci_project_flavors_zone_tooltip_link' | translate }}"
+                                        target="_blank"
+                                        data-translate="pci_project_flavors_zone_tooltip_link"
+                                    ></a>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html:133]: **nitpick:** Link should be in the same line.
```suggestion
                                  <p class="text-center">
                                      <span
                                        data-translate="pci_project_flavors_zone_localzone_tooltip"
                                    ></span>
                                    <a
                                        class="fa fa-external-link ml-2"
                                        aria-hidden="true"
                                        data-ng-href="{{$ctrl.localZoneUrl}}"
                                        title="{{ 'pci_project_flavors_zone_tooltip_link' | translate }}"
                                        target="_blank"
                                        data-translate="pci_project_flavors_zone_tooltip_link"
                                    ></a>
                                 </p>
```

File: packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html [hunk 4]
```diff
@@ -103,14 +103,68 @@
...
+                                        data-translate="pci_project_flavors_zone_globalregions_tooltip"
+                                    ></p>
+                                    <a
+                                        class="fa fa-external-link ml-2"
+                                        aria-hidden="true"
+                                        data-ng-href="{{$ctrl.globalRegionsUrl}}"
+                                        title="{{ 'pci_project_flavors_zone_tooltip_link' | translate }}"
+                                        target="_blank"
+                                        data-translate="pci_project_flavors_zone_tooltip_link"
+                                    ></a>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html:165]: Same as https://github.com/ovh/manager/pull/10606/files#r1441388361

File: packages/manager/modules/pci/src/projects/project/instances/add/add.html [hunk 5]
```diff
@@ -70,15 +70,69 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
...
+                                                data-translate="pci_project_flavors_zone_localzone_tooltip"
+                                            ></p>
+                                            <a
+                                                class="fa fa-external-link ml-2"
+                                                aria-hidden="true"
+                                                data-ng-href="{{$ctrl.localZoneUrl}}"
+                                                title="{{ 'pci_project_flavors_zone_tooltip_link' | translate }}"
+                                                target="_blank"
+                                                data-translate="pci_project_flavors_zone_tooltip_link"
+                                            ></a>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.html:100]: Same as https://github.com/ovh/manager/pull/10606/files#r1441388361

File: packages/manager/modules/pci/src/projects/project/project.service.js [hunk 6]
```diff
@@ -1,15 +1,26 @@
...
     this.$http = $http;
     this.iceberg = iceberg;
+    this.coreConfig = coreConfig;
   }
 
   getProjectInfo() {
     return this.project;
   }
 
+  getDocUrl(SUFFIX) {
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/project.service.js:16]: **nitpick:** Dont use short names
```suggestion
  getDocumentUrl(suffix) {
```

File: packages/manager/modules/pci/src/projects/project/project.service.js (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/project.service.js:22]: yup

File: packages/manager/modules/pci/src/projects/project/project.constants.js (hunk 2 above)
Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/project.constants.js:176]: Will check this with Mounir as of now in the list the prefix is same.

File: packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html (hunk 3 above)
Comment (sachinrameshn) [reply packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html:133]: In the mock ups link is given in new line.

File: packages/manager/modules/pci/src/projects/project/project.service.js (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/project.service.js]: I have addressed all your comments

**Pull request merged**